### PR TITLE
Integrate Frgaal in Java libs

### PR DESCRIPTION
**Pull request by hubertp**
[ci no changelog needed]

### Pull Request Description

This PR allows for easier integration of Frgaal in different subprojects.
Related to https://www.pivotaltracker.com/story/show/182093808

### Checklist

Please include the following checklist in your PR:

- [ ] The documentation has been updated if necessary.
- [x] All code conforms to the [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md), [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md), and [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guides.
- All code has been tested:
  - [ ] Unit tests have been written where possible.
  - [ ] If GUI codebase was changed: Enso GUI was tested when built using BOTH `./run dist` and `./run watch`.

### CODE REVIEW

**Comment by 4e6** [comment]
@hubertp there are some issues with compilation https://github.com/enso-org/enso/runs/6320685106?check_suite_focus=true#step:4:298

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/enso-org/enso">Integrate Frgaal in Java libs · enso-org/enso@b70ec8e</a></strong></div><div>Hybrid visual and textual functional programming. Contribute to enso-org/enso development by creating an account on GitHub.</div></blockquote>

**Comment by hubertp** [comment]
@4e6 thanks, I somehow missed it. Sorry

**Comment by hubertp** [comment]
@4e6 done

File: build.sbt [hunk 1]
```diff
@@ -1589,14 +1597,16 @@ lazy val `std-base` = project
...
     autoScalaLibrary := false,
     Compile / packageBin / artifactPath :=
       `table-polyglot-root` / "std-table.jar",
     libraryDependencies ++= Seq(
       "com.ibm.icu"         % "icu4j"             % icuVersion,
       "com.univocity"       % "univocity-parsers" % "2.9.0",
       "org.apache.poi"      % "poi-ooxml"         % "5.0.0",
-      "org.apache.xmlbeans" % "xmlbeans"          % "5.0.1"
+      "org.apache.xmlbeans" % "xmlbeans"          % "5.0.1",
+      "org.graalvm.truffle" % "truffle-api"       % graalVersion % "provided"
```
Comment (4e6) [inline build.sbt:1648]: I wonder how it worked before

**Review by 4e6 — APPROVED** [review]
Looks good!

**Pull request merged**